app.py

```diff
@@ -21,4 +26,19 @@ async def add_role(member: discord.Member, role_id: int):
     await member.add_roles(role)
 
 
+async def set_welcome_message():
+    channel: discord.TextChannel = bot.get_channel(876672670903717948)
+    message = await channel.fetch_message(880000637998338079)
```

Comment:
Hard-coded channel ids for now. I'll move these over to `LacusConfig` if this is a good approach.